sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -1510,33 +1510,23 @@ internal static LinkedEntityMatch LinkedEntityMatch(double confidenceScore = def
         }
         #endregion Healthcare
 
-        // added for compilation
-        /// <summary>
-        /// added for compilation
-        /// </summary>
-        /// <returns></returns>
-        public static HealthcareEntityAssertion HealthcareEntityAssertion()
+        /// <summary> Initializes a new instance of HealthcareEntityAssertion. </summary>
```

Comment:
@AlexanderSher do you know why the generator is not creating the TextAnalytictsModelFactory anymore? (before it was [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/TextAnalyticsModelFactory.cs))